inlong-sdk/transform-sdk/src/test/java/org/apache/inlong/sdk/transform/process/TestTransformArithmeticFunctionsProcessor.java

```diff
@@ -55,6 +55,39 @@ public class TestTransformArithmeticFunctionsProcessor {
         kvSink = new KvSinkInfo("UTF-8", dstFields);
     }
 
+    @Test
+    public void testMd5Function() throws Exception {
+        String transformSql = "select md5(numeric1) from source";
+        TransformConfig config = new TransformConfig(transformSql);
+        TransformProcessor<String, String> processor = TransformProcessor
+                .create(config, SourceDecoderFactory.createCsvDecoder(csvSource),
+                        SinkEncoderFactory.createKvEncoder(kvSink));
+        // case1: "1|4|6|8"
+        List<String> output1 = processor.transform("1|4|6|8");
+        Assert.assertEquals(1, output1.size());
+        Assert.assertEquals("result=c4ca4238a0b923820dcc509a6f75849b", output1.get(0));
+
+        // case2: "-1|4|6|8"
+        List<String> output2 = processor.transform("-1|4|6|8");
+        Assert.assertEquals(1, output2.size());
+        Assert.assertEquals("result=6bb61e3b7bce0931da574d19d1d82c88", output2.get(0));
+
+        // case3: "|4|6|8"
+        List<String> output3 = processor.transform("|4|6|8");
+        Assert.assertEquals(1, output3.size());
+        Assert.assertEquals("result=null", output3.get(0));
+
+        // case4: "1|4|6|8"
+        transformSql = "select md5(numericxx) from source";
+        config = new TransformConfig(transformSql);
+        processor = TransformProcessor
+                .create(config, SourceDecoderFactory.createCsvDecoder(csvSource),
+                        SinkEncoderFactory.createKvEncoder(kvSink));
+        List<String> output4 = processor.transform("1|4|6|8");
+        Assert.assertEquals(1, output4.size());
+        Assert.assertEquals("result=null", output4.get(0));
```

Comment:
It is better to add a test case: md5("").

Reply:
done